modules/oe_list_pages_open_vocabularies/tests/src/FunctionalJavascript/OpenVocabulariesFiltersTest.php

```diff
@@ -0,0 +1,370 @@
+      'fields' => $fields,
+    ];
+
+    $this->container->get('kernel')->rebuildContainer();
+    /** @var \Drupal\open_vocabularies\OpenVocabularyAssociationInterface $association */
+    $association = OpenVocabularyAssociation::create($values);
+    $association->save();
+    $association_id = $association->id();
+    $this->association = $association;
+
+    // Create some taxonomy terms.
+    $values = [
+      'name' => 'yellow color',
+      'vid' => 'vocabulary_one',
+    ];
+    $term_1 = Term::create($values);
+    $term_1->save();
+
+    $values = [
+      'name' => 'green color',
+      'vid' => 'vocabulary_one',
+    ];
+    $term_2 = Term::create($values);
+    $term_2->save();
+
+    // Create some test nodes to index and search in.
+    $date = new DrupalDateTime('09-02-2021');
+    $values = [
+      'title' => 'one yellow fruit',
+      'type' => 'content_type_one',
+      'body' => 'this is a banana',
+      'status' => NodeInterface::PUBLISHED,
+      'field_select_one' => 'test1',
+      'field_open_vocabularies' => [
+        'target_id' => $term_1->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+    $values = [
+      'title' => 'a green fruit',
+      'type' => 'content_type_one',
+      'body' => 'this is an avocado',
+      'status' => NodeInterface::PUBLISHED,
+      'field_select_one' => 'test2',
+      'field_open_vocabularies' => [
+        'target_id' => $term_2->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->modify('+ 5 days')->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    // Create some test nodes to index and search in.
+    $date = new DrupalDateTime('09-02-2021');
+    $values = [
+      'title' => 'Sun',
+      'type' => 'content_type_two',
+      'body' => 'This is the sun',
+      'status' => NodeInterface::PUBLISHED,
+      'field_open_vocabularies' => [
+        'target_id' => $term_1->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    $values = [
+      'title' => 'Grass',
+      'type' => 'content_type_two',
+      'body' => 'This is grass',
+      'status' => NodeInterface::PUBLISHED,
+      'field_open_vocabularies' => [
+        'target_id' => $term_2->id(),
+        'target_association_id' => $association_id,
+      ],
+      'created' => $date->modify('+ 5 days')->getTimestamp(),
+    ];
+    $node = Node::create($values);
+    $node->save();
+
+    /** @var \Drupal\search_api\Entity\Index $index */
+    $index = Index::load('node');
+    $index->indexItems();
+  }
+
+  /**
+   * Test exposed filters and default filters configuration.
+   */
+  public function testListPagePluginFiltersFormConfiguration(): void {
```

Comment:
Mmm, this is the only functional test we have of open vocabs. Why aren't we covering having two different associations? That's a critical part of the system the fact that for each association we generate a facet and index field.